packages/react-tabs/Spec.md

```diff
@@ -213,18 +229,20 @@ Handling overflow is a built-in feature of tabs and replaceable via a slot.
 
 ### Tab
 
-- TabCommons.value supports an arbitrary identifier for each tab that can be paired with a future tab panel. This is similar to Accordian's selection approach.
+TabCommons.value supports an arbitrary identifier value.
+The value for each tab can be used to display the associated tab panel (future).
+This is similar to Accordian's selection approach.
+
+> TODO: Should value be called key since the tab list has selectedKey?
+> If so, should this key be applied to the tab as a key attribute?
```

Comment:
If by future we mean after the implementation of Tabs, can we perhaps remove this and only add it when we have a need for it?